pulsar-broker/src/test/java/org/apache/pulsar/client/impl/MessageChunkingTest.java

```diff
@@ -348,6 +363,10 @@ public void testMaxPendingChunkMessages() throws Exception {
         assertEquals(receivedMsg.getValue(), "chunk-1-0|chunk-1-1|");
 
         consumer.acknowledge(receivedMsg);
+        assertEquals(admin.topics().getStats(topicName).getSubscriptions().get(subName)
+                .getNonContiguousDeletedMessagesRanges(), 0);
```

Comment:
Need to remove this line?

Reply:
This line is used to fix this flaky test.

Reply:
https://github.com/apache/pulsar/pull/21103/files#diff-9ce7ee7d161127cb26c5c43598bc3cf93021123b69a5f8669c988bdb2a36b795R368-R369 will fixed the flaky test?